### ISSUE TITLE
bug: unable to cancel running flow

### ISSUE THREAD

**Issue by mrl5** [windmill-labs/windmill#785]
### Describe the bug

I'm unable to cancel running flow

### To reproduce

1. go to https://app.windmill.dev/scripts and create this void script:
```typescript
export async function main(arg: string) {
	return { arg }
}
```
2. go to https://app.windmill.dev/flows
3. add new flow
4. create 1st flow step by clicking on "pick script from your workspace" and select script created in `#1`
5. switch to "sleep/suspend" and check "sleep after module successful execution" -> set e.g. 30 seconds
6. create 2nd flow step by replicating `#4`
7. save flow
8. run flow
9. try to cancel flow after first step executes

### actual result
is that flow can't be cancelled although I got HTTP 200 for `/api/w/demo/jobs/queue/cancel/0183f75c-f56d-77f4-56c4-78c1b45da513` + toast message in UI indicating that flow was cancelled

### Expected behavior

flow can be cancelled

### Additional Context

flow can be cancelled when you're testing the flow (before saving)

**Comment by rubenfiszel** [comment]
Should be fixed by: cb5ed9b , please lmk if it didn't

**Comment by mrl5** [comment]
this is indeed fixed now. after clicking "cancel" flow is eventually canceled when sleep time is over
